src/payment/checkout/payment-form/StripePaymentForm.jsx

```diff
@@ -123,6 +123,8 @@ function StripePaymentForm({
         // Show error in payment form
         if (result.error.type === 'card_error' || result.error.type === 'validation_error') {
           setMessage(result.error.message);
+        } else if (result.error.type === 'invalid_request_error' && result.error.code === 'payment_intent_unexpected_state') {
```

Comment:
This if/else block is for showing a message to the user in the Stripe form (at the bottom). Is this what we want here on the new code added? If yes, can you update the comment on L123

Reply:
`setMessage` will show the message in the Stripe element, whereas the saga dispatch in this if/else block will show a banner to the user. Would you like me to remove the comment?